src/storage/secondary/rowset/raw.rs

```diff
@@ -0,0 +1,34 @@
+use std::sync::Arc;
+
+use crate::catalog::ColumnCatalog;
+
+/// Raw columns.
```

Comment:
What's the meaning of `Raw`? Better to name it as `EncodedColumn` or something?

Reply:
Intended to be "in raw bytes". "Encoded" is better as with different encoders we get different bytes.
I'll fix it.